client/src/components/Tickets/TicketModal.scss

```diff
@@ -102,6 +102,7 @@
   }
   .link {
```

Comment:
**Usability.** [FP-1339: Ticket Attachment Cell Hover Underlines When Not on Link](https://jira.tacc.utexas.edu/browse/FP-1339)

I hope it can be fixed, at the latest, before the feature launches.

Reply:
Note: [Sal's request to shorten the length of the "Download" links' cells](https://github.com/TACC/Core-Portal/pull/542#discussion_r748355154) will _reduce_, but _**not** fix_, this problem.

Reply:
@williamkwon97 @tacc-wbomar This may need to be done as a follow on task: https://jira.tacc.utexas.edu/browse/FP-1339

Reply:
👍